docs/code/IDataViewImplementation.md

```diff
@@ -383,15 +384,15 @@ inconsistency, surprises and bugs for users and developers.
 
 # Thread Safety
 
-Any `IDataView` implementation, as well as the `ISchema`, *must* be thread
+Any `IDataView` implementation, as well as the `DataViewSchema`, *must* be thread
```

Comment:
I see some remaining occurrences of `ISchema` in comments:
* https://github.com/dotnet/machinelearning/blob/beb67a128544ee98b8c4309859f7e5a9f679ac4e/src/Microsoft.ML.Data/Transforms/TransformBase.cs#L257
* https://github.com/dotnet/machinelearning/blob/70ef7ecd43b031b481a4047ea361da5e2f360336/src/Microsoft.ML.Core/Data/ISchemaBindableMapper.cs#L12

Reply:
I have now also updated the comments outside of the "docs/code" directory. #Resolved